pysteps/feature/tstorm.py

```diff
@@ -0,0 +1,320 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+pysteps.feature.tstorm
+======================
+
+Thunderstorm cell detection module, part of Thunderstorm Detection and Tracking (DATing)
+This module was implemented following the procedures used in the TRT Thunderstorms
+Radar Tracking algorithm (:cite:`TRT2004`) used operationally at MeteoSwiss.
+Modifications include advecting the identified thunderstorms with the optical flow
+obtained from pysteps, as well as additional options in the thresholding.
+
+References
+...............
+:cite:`TRT2004`
+
+@author: mfeldman
+
+.. autosummary::
+    :toctree: ../generated/
+
+    detection
+    belonging
```

Comment:
In the ReadTheDocs, there is no description of 'belonging'. The other three (detection, longdistance and get_profile) do have a description.

Reply:
8ad3af4 should have fixed this